docs/sdks/uid2-sdk-ref-ios.md

```diff
@@ -91,27 +95,60 @@ Install the iOS SDK via Swift Package Manager (SPM). There are two installation
 
 Add the following dependency to Package.swift:
 
-``` javascript
+```js
 dependencies: [
-.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "0.3.0"),
+.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "1.2.0"),
```

Comment:
i dont think there is a swift specific code markup

Reply:
will ask @dcaunt on this, currently it's rendered as:
![image](https://github.com/IABTechLab/uid2docs/assets/2499852/74a07c46-83e3-4b24-97d8-a27f3c3a651d)

Reply:
thanks @dcaunt fixed per your comment. the highlighter is strange - as we use none of the ones listed (js instead of jsx"

will leave it for now tho as changing to swift/kotlin currently doesnt change the rendering. for future investigation maybe